src/components/TodoList/TodoList.tsx

```diff
@@ -1,14 +1,20 @@
 /* eslint-disable */
 import React from 'react';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => {
+import { TodoItem } from '../TodoItem';
+type Props = {
+  todos: Todo[];
+};
+export const TodoList: React.FC<Props> = ({ todos }) => {
   return (
     <>
-      <p className="notification is-warning">
-        There are no todos matching current filter criteria
-      </p>
-
-      <table className="table is-narrow is-fullwidth">
+      {!todos.length ? (
+        <p className="notification is-warning">
+          There are no todos matching current filter criteria
+        </p>
```

Comment:
You should do it in the parent component as `TodoList` just get and render todos